glotaran/parameter/parameter.py

```diff
@@ -334,7 +334,7 @@ def __repr__(self):
 
     def __array__(self):
         """array"""
-        return np.array([float(self._getval())])
+        return np.array(float(self._getval()), dtype=float)
```

Comment:
Note for posterity: this was the essential change necessary to make the code work with numpy 1.20